test/lib/absinthe/type/custom_test.exs

```diff
@@ -37,6 +38,11 @@ defmodule Absinthe.Type.CustomTest do
       assert {:ok, @datetime} == parse(:datetime, "2017-01-27 20:31:55Z")
     end
 
+    it "cannot be parsed when a UTC offset is included" do
```

Comment:
Sorry to be pedantic, but I think should should be "non-zero UTC offset". I don't believe this string will not error:

`2017-01-27T20:31:55+00:00"`